addon/README.md

```diff
@@ -16,7 +16,7 @@ dev: [dev-prefs.json](dev-prefs.json)
 
 You may need to set both `xpinstall.signatures.required` and
 `xpinstall.whitelist.required` flags to false in `about:config`
-if the addon has not yet been signed.
+if the add-on has not yet been signed.
```

Comment:
Do we want to remove this section now that the ~~addon~~ add-on is signed, or is it still relevant for local development?